# WarehouseMap.msg

# Lista de estaciones de carga y descarga (2D - [x, y])
float32[] stations

# Lista de estanterÃ­as (2D - [x, y])
float32[] shelves

# Lista de muros (2D - [x, y])
float32[] walls
